### PR TITLE
JAMES-2526 Public webadmin routes

### PR DESCRIPTION
**Pull request by chibenwa**
Review only last commit

### CODE REVIEW

**Comment by chibenwa** [comment]
```
GroupMappingRelayTest.lambda$sendMessageShouldSendAMessageToAnExternalGroupMember$1:160->lambda$null$0:161 1 expectation failed.
[44f2f1bdf5c3c4be03df80a01fcf94492d675610] JSON path [0].from doesn't match.
[44f2f1bdf5c3c4be03df80a01fcf94492d675610] Expected: fromuser@domain1.com
[44f2f1bdf5c3c4be03df80a01fcf94492d675610]   Actual: null
```

**Comment by aduprat** [comment]
Rebase on master + fixup on grouping authentication tests.

**Comment by trantienduchn** [comment]
```
[2d0be743530e1a06dd2a337999a848ca4f76b233] [INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.432 s - in org.apache.james.webadmin.integration.AuthorizedEndpointsTest
[2d0be743530e1a06dd2a337999a848ca4f76b233] [INFO] 
[2d0be743530e1a06dd2a337999a848ca4f76b233] [INFO] Results:
[2d0be743530e1a06dd2a337999a848ca4f76b233] [INFO] 
[2d0be743530e1a06dd2a337999a848ca4f76b233] [ERROR] Errors: 
[2d0be743530e1a06dd2a337999a848ca4f76b233] [ERROR]   UnauthorizedEndpointsTest » NoClassDefFound org/apache/james/spamassassin/Spam...
[2d0be743530e1a06dd2a337999a848ca4f76b233] [INFO] 
[2d0be743530e1a06dd2a337999a848ca4f76b233] [ERROR] Tests run: 23, Failures: 0, Errors: 1, Skipped: 0
[2d0be743530e1a06dd2a337999a848ca4f76b233] [INFO] 
[2d0be743530e1a06dd2a337999a848ca4f76b233] [INFO] ------------------------------------------------------------------------
[2d0be743530e1a06dd2a337999a848ca4f76b233] [INFO] Reactor Summary:
```

File: server/protocols/webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/UnauthorizedEndpointsTest.java [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+    @BeforeEach
+    void setUp(JamesWithSpamAssassin james) throws Throwable {
+        WebAdminGuiceProbe webAdminGuiceProbe = james.getJmapServer().getProbe(WebAdminGuiceProbe.class);
+
+        RestAssured.requestSpecification = WebAdminUtils.buildRequestSpecification(webAdminGuiceProbe.getWebAdminPort())
+            .build();
+    }
+
+    @AfterEach
+    void tearDown(JamesWithSpamAssassin james) {
```
Comment (chibenwa) [inline server/protocols/webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/UnauthorizedEndpointsTest.java]: Is it compulsory to get SpamAssassin here?

**Comment by aduprat** [comment]
```
[ERROR] Errors: 
[ERROR]   RabbitMQClusterTest$ClusterNodesFailure.setup:165 » IO
```
test this please

**Comment by rouazana** [comment]
again
```
[32ffce7ed7bf6918a4aac23916054e7377e9ac3e] [ERROR] Tests run: 6, Failures: 0, Errors: 1, Skipped: 1, Time elapsed: 138.695 s <<< FAILURE! - in org.apache.james.queue.rabbitmq.RabbitMQClusterTest
[32ffce7ed7bf6918a4aac23916054e7377e9ac3e] [ERROR] nodeKillingWhenConsuming{DockerRabbitMQCluster}  Time elapsed: 35.938 s  <<< ERROR!
[32ffce7ed7bf6918a4aac23916054e7377e9ac3e] java.io.IOException
[32ffce7ed7bf6918a4aac23916054e7377e9ac3e] 	at org.apache.james.queue.rabbitmq.RabbitMQClusterTest$ClusterNodesFailure.setup(RabbitMQClusterTest.java:165)
[32ffce7ed7bf6918a4aac23916054e7377e9ac3e] Caused by: com.rabbitmq.client.ShutdownSignalException: connection error
[32ffce7ed7bf6918a4aac23916054e7377e9ac3e] 	at org.apache.james.queue.rabbitmq.RabbitMQClusterTest$ClusterNodesFailure.setup(RabbitMQClusterTest.java:165)
[32ffce7ed7bf6918a4aac23916054e7377e9ac3e] Caused by: java.net.SocketException: Connection reset
```
test this please

**Comment by chibenwa** [comment]
```
[32ffce7ed7bf6918a4aac23916054e7377e9ac3e] + docker run --detach=true --name=james-server-jpa-james-2526-with-auth-08f32ef6-0875-4759-8a2f-d91fbbb2df88 --hostname james.linagora.com --expose=143 --publish-all=true james-server-jpa-james-2526-with-auth-08f32ef6-0875-4759-8a2f-d91fbbb2df88
[32ffce7ed7bf6918a4aac23916054e7377e9ac3e] 29fdff66d65f84bd6710b016889ac615fd4d4e82f148a780f40af5dadbcf9a07
[32ffce7ed7bf6918a4aac23916054e7377e9ac3e] Running: Print Message
[32ffce7ed7bf6918a4aac23916054e7377e9ac3e] Waiting for James server to be deployed
[32ffce7ed7bf6918a4aac23916054e7377e9ac3e] Running: Shell Script
```

Failed to deploy James JPA guice apparently

**Comment by aduprat** [comment]
test this please

**Comment by aduprat** [comment]
squashed & rebased

**Pull request merged**